pkgs/development/compilers/ghc/9.2.4-binary.nix

```
@@ -236,6 +236,7 @@ stdenv.mkDerivation rec {
     ''
       patchShebangs ghc-${version}/utils/
       patchShebangs ghc-${version}/configure
+      patchShebangs ghc-${version}/inplace/bin
```

Sorry, I should have been clearer here. In some situation means that this directory doesn't exist on some platforms (i.e. x86_64-darwin), so it may only be executed conditionally.

```suggestion
      test -d ghc-${version}/inplace/bin && \
        patchShebangs ghc-${version}/inplace/bin
```